src/test/kotlin/net/lumalyte/lg/infrastructure/persistence/guilds/GuildRepositoryBannermanTest.kt

```
@@ -0,0 +1,258 @@
+        connection.createStatement().use { stmt ->
+            stmt.executeQuery("SELECT * FROM guilds WHERE id = '$guildId'").use { rs ->
+                if (!rs.next()) return null
+
+                val name = rs.getString("name")
+                val level = rs.getInt("level")
+                val bankBalance = rs.getInt("bank_balance")
+                val mode = GuildMode.valueOf(rs.getString("mode").uppercase())
+                val createdAtStr = rs.getString("created_at")
+                val createdAt = try {
+                    Instant.parse(createdAtStr)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)